### PR TITLE
Seeds now support multiple seed directories, with a sorting option.

### PR DESCRIPTION
**Pull request by candyapplecorn**
### This pull request enables Knex's seed feature to:

- Create seeds in specified subdirectories.
- Run seeds from multiple subdirectories, with a sorting option.

Link to **documentation** branch's pull request: https://github.com/knex/documentation/pull/220

### CODE REVIEW

**Comment by candyapplecorn** [comment]
Closed because actual manual testing revealed this code to not work. Thought the jake cli tests would take care of that for me but nope, there's more to it.

File: test/jake/jakelib/seed-test.js [hunk 1]
```diff
@@ -15,6 +15,75 @@ const KNEX = path.normalize(__dirname + '/../../../bin/cli.js');
 const taskList = [];
 /* * * TESTS * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
+test(
```
Comment (kibertoad) [inline test/jake/jakelib/seed-test.js]: I would say that for new tests `cli-testlab` tests would be preferable (and easier to write) than using jake directly. See test/cli/migrate-make.spec.js for an example.

Also async/await is preferred over using promise chains.

File: test/jake-util/seeds-in-subdirs/production/first.js [hunk 2]
```diff
@@ -0,0 +1 @@
+exports.seed = (knex, Promise) => {};
```
Comment (kibertoad) [inline test/jake-util/seeds-in-subdirs/production/first.js]: Promise is not passed to seeds anymore, I think

File: test/jake/jakelib/seed-test.js (hunk 1 above)
Comment (candyapplecorn) [reply test/jake/jakelib/seed-test.js]: @kibertoad Thanks for the tips. I will use cli-testlab instead. Also, yeah, I definitely didn't want to be using promise chains and Bluebird. I just figured the rest of the codebase was using it, so I should too. So, cool! async await valhalla awaits me

**Comment by kibertoad** [comment]
Released in 0.21.3

**Pull request closed without merging**